tests/test_the_test/test_remote_config.py

```diff
@@ -4,7 +4,7 @@
 @scenarios.test_the_test
 def test_debugger_command_none():
     expected = {
-        "targets": "eyJzaWduZWQiOiB7Il90eXBlIjogInRhcmdldHMiLCAiY3VzdG9tIjogeyJvcGFxdWVfYmFja2VuZF9zdGF0ZSI6ICJleUptYjI4aU9pQWlZbUZ5SW4wPSJ9LCAiZXhwaXJlcyI6ICIzMDAwLTAxLTAxVDAwOjAwOjAwWiIsICJzcGVjX3ZlcnNpb24iOiAiMS4wIiwgInRhcmdldHMiOiB7fSwgInZlcnNpb24iOiAwfSwgInNpZ25hdHVyZXMiOiBbeyJrZXlpZCI6ICJlZDc2NzJjOWEyNGFiZGE3ODg3MmVlMzJlZTcxYzdjYjFkNTIzNWU4ZGI0ZWNiZjFjYTI4YjljNTBlYjc1ZDllIiwgInNpZyI6ICJlMjI3OWE1NTRkNTI1MDNmNWJkNjhlMGE5OTEwYzdlOTBjOWJiODE3NDRmZTljODgyNGVhMzczN2IyNzlkOWU2OWIzY2U1ZjRiNDYzYzQwMmViZTM0OTY0ZmI3YTY5NjI1ZWIwZTkxZDNkZGJkMzkyY2M4YjMyMTAzNzNkOWIwZiJ9XX0=",
+        "targets": "ewogICJzaWduZWQiOiB7CiAgICAiX3R5cGUiOiAidGFyZ2V0cyIsCiAgICAiY3VzdG9tIjogewogICAgICAib3BhcXVlX2JhY2tlbmRfc3RhdGUiOiAiZXlKbWIyOGlPaUFpWW1GeUluMD0iCiAgICB9LAogICAgImV4cGlyZXMiOiAiMzAwMC0wMS0wMVQwMDowMDowMFoiLAogICAgInNwZWNfdmVyc2lvbiI6ICIxLjAiLAogICAgInRhcmdldHMiOiB7fSwKICAgICJ2ZXJzaW9uIjogMAogIH0sCiAgInNpZ25hdHVyZXMiOiBbCiAgICB7CiAgICAgICJrZXlpZCI6ICJlZDc2NzJjOWEyNGFiZGE3ODg3MmVlMzJlZTcxYzdjYjFkNTIzNWU4ZGI0ZWNiZjFjYTI4YjljNTBlYjc1ZDllIiwKICAgICAgInNpZyI6ICJlMjI3OWE1NTRkNTI1MDNmNWJkNjhlMGE5OTEwYzdlOTBjOWJiODE3NDRmZTljODgyNGVhMzczN2IyNzlkOWU2OWIzY2U1ZjRiNDYzYzQwMmViZTM0OTY0ZmI3YTY5NjI1ZWIwZTkxZDNkZGJkMzkyY2M4YjMyMTAzNzNkOWIwZiIKICAgIH0KICBdCn0=",
```

Comment:
You could consider matching the base64 decoded instead to make the expectations human-readable

Reply:
You have a point, I'll consider that, but not in this PR. I want to refactor all remote config tests related to debugger, and this test probably will be dropped.